src/services/codefixes/removeAccidentalCallParentheses.ts

```diff
@@ -0,0 +1,21 @@
+/* @internal */
+namespace ts.codefix {
+    const fixId = "removeAccidentalCallParentheses";
+    const errorCodes = [
+        Diagnostics.This_expression_is_not_callable_because_it_is_a_get_accessor_Did_you_mean_to_use_it_without.code,
+    ];
+    registerCodeFix({
+        errorCodes,
+        getCodeActions(context) {
+            const callExpression = findAncestor(getTokenAtPosition(context.sourceFile, context.span.start), isCallExpression);
+            if (!callExpression) {
+                return undefined;
+            }
+            const changes = textChanges.ChangeTracker.with(context, t => {
+                t.deleteNodeRange(context.sourceFile, callExpression.openParenToken, callExpression.closeParenToken);

```

Comment:
Instead of adding tokens to the tree, why not just erase the text from `callExpression.expression.end` to `callExpression.end`?

Reply:
Good idea, I didn't think of that!

Reply:
@DanielRosenwasser would you mind taking another look when you get a chance? Thanks!